src/js/main.js

```diff
@@ -728,6 +728,17 @@ Site.Minimap = {
     var x = col * Site.Map.window.width * -1;
     var y = row * Site.Map.window.height * -1;
 
+    var $map = $('#map');
```

Comment:
In the same object the map is addressed via vanilla js and not cached. Would therefore make sense to refactor and use jquery throughout and cache from the start.